src/main/java/org/spongepowered/api/item/merchant/TradeOffer.java

```diff
@@ -186,6 +190,24 @@ static Builder builder() {
          */
         Builder canGrantExperience(boolean experience);
 
+        /**
+         * Sets how much experience the trade offer will grant upon use.
+         *
+         * @param experience The amount of experience.
+         * @return This builder
+         */
+        Builder grantsExperience(int experience);
+
+        /**
+         * Sets the rate at which this trade offer's price will grow by when used.
+         *
+         * <p>By default this multiplier is 0.05</p>
```

Comment:
If you're going to link modifiers like this, complete the sentence, and possibly link the MC wiki where they explain this.